src/util/util.js

```diff
@@ -239,6 +239,19 @@ export function wrap(n: number, min: number, max: number): number {
     return (w === min) ? max : w;
 }
 
+/**
+ * Computes shortest angle in range [-180, 180) between two angles.
+ *
+ * @param {*} a First angle in degrees
+ * @param {*} b Second angle in degrees
+ * @returns Shortest angle
+ * @private
+ */
+export function shortestAngle(a: number, b: number): number {
+    const diff = (b - a + 180) % 360 - 180;
+    return diff < -180 ? diff + 360 : diff;
+}
```

Comment:
Why is this function in `util.js` rather than inlined? Will it be used in other places?

Reply:
Finding signed distance between two angles is a quite common and useful operation and a good candidate for having its own utility function. I couldn't find any good reasons for inlining it as the code path is not particularly hot and it's easier to test too. Would you like to see it inlined instead?

Reply:
Those all seem like good reasons. Great work @mpulkki-mapbox!